unitTests/ui/DynamicFacet/DynamicFacetTest.ts

```diff
@@ -341,5 +341,24 @@ export function DynamicFacetTest() {
 
       expect(test.cmp.analyticsFacetState).toEqual([test.cmp.values.get('bar').analyticsMeta, test.cmp.values.get('foo').analyticsMeta]);
     });
```

Comment:
Should there be tests for `putStateIntoQueryBuilder` and `logAnalyticsEvent` considering they are `public`?

Reply:
putStateIntoQueryBuilder only calls one function on the dynamicFacetQueryController, which is private and thoroughly tested already.
logAnalyticsEvent is already tested (previous PR)